pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -39,8 +38,11 @@ const (
 	maxRetries = 5
 )
 
-// Required for unit test override.
-var loadKubeConfig = clientcmd.Load
+// LoadKubeConfig is a unit test override variable for loading the k8s config.
```

Comment:
Should we add some explicit DO NOT USE -- TEST ONLY comments here?

Reply:
sure. i'll add it here and the variable below.